tests/lib/prepare-restore.sh

```diff
@@ -608,6 +608,15 @@ prepare_project() {
         disable_journald_rate_limiting
         disable_journald_start_limiting
     fi
+
+    # native jq replacement, but still with some incompatibilies, see
+    # https://github.com/itchyny/gojq
+    # major differences:
+    # - map keys are sorted by default
+    # - with --yaml-input, can parse YAML
+    GOBIN=$PROJECT_PATH/tests/bin \
+    CGO_ENABLED=0 \
+        go install github.com/itchyny/gojq/cmd/gojq@v0.12.16
```

Comment:
@bboozzoo what if we create a gojq snap and then we use an alias to keep using jq command?
If we do that, we don't need to update the tests.